tests/ludwig/models/modules/encoder_test.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import tensorflow as tf
+
+from ludwig.models.modules.image_encoders import Stacked2DCNN, ResNetEncoder
+from ludwig.models.modules.loss_modules import regularizer_registry
+
+
+L1_REGULARIZER = regularizer_registry['l1'](0.1)
+L2_REGULARIZER = regularizer_registry['l2'](0.1)
+NO_REGULARIZER = None
+DROPOUT_RATE = 0.5
+
+
+def encoder_test(encoder_type,
```

Comment:
Let's stick to this format - most of the code base is like this:

def func(
    arg_1,
    arg_2
):